src/scenic/simulators/robosuite/model.scenic

```diff
@@ -0,0 +1,253 @@
+# src/scenic/simulators/robosuite/model.scenic
+
+"""Scenic world model for RoboSuite - Custom Environments Only."""
+
+from .simulator import RobosuiteSimulator, SetJointPositions, OSCPositionAction
+
+# Global parameters with defaults matching Robosuite's defaults
+param env_config = {}
```

Comment:
Please add documentation for these global parameters to the module docstring (see `scenic.simulators.carla.model` for an example).